server/src/database/config/build.sql

```diff
@@ -11,7 +11,8 @@ CREATE TABLE users (
   phoneNumber VARCHAR(255),
   password VARCHAR(255) NOT NULL,
   birthDate DATE,
-  bio VARCHAR(255)
+  bio VARCHAR(255),
+  isVerified BOOLEAN DEFAULT FALSE,
```

Comment:
User can not created in database if he is not verified, so why you add this property?

Reply:
Okay, Nizar,
I forgot this point.